worker/snapshot_test.go

```diff
@@ -81,7 +83,10 @@ func TestSnapshot(t *testing.T) {
 	require.NoError(t, err)
 
 	// Wait for the container to start.
-	time.Sleep(time.Second * 2)
+	err = testutil.CheckHealthContainer(testutil.ContainerAddr("alpha2", 8080))
+	if err != nil {
+		fmt.Print("error while getting alpha container health")
```

Comment:
this should fail the test with the same error

Reply:
done